backend/src/db/migrations/20241024194046_secret-query-indexes.ts

```diff
@@ -0,0 +1,39 @@
+import { Knex } from "knex";
+
+import { TableName } from "@app/db/schemas";
+
+// add indexes to improve dashboard search performance
+
+export async function up(knex: Knex): Promise<void> {
```

Comment:
Did you an `Explain` over the folder dal operations to ensure this will boost the performance

Reply:
not being used; removed